corehq/apps/es/management/commands/verify_reindex.py

```diff
@@ -1,26 +1,27 @@
 import argparse
 import re
 from textwrap import dedent
+from django.conf import settings
 
 from django.core.management.base import BaseCommand
 
 
 class Command(BaseCommand):
-    help = dedent("""\
+    help = dedent(f"""\
     Verify if a reindex process is completed in a ElasticSearch 2 cluster.
     The command parses the logs for reindex and provides detailed information of the process.
 
     The log should be extracted from elasticsearch logs. It can be done using commcare cloud.
     ```
-    cchq <env> run-shell-command elasticsearch "cat /opt/data/elasticsearch*/logs/*es.log | grep <task_id>"
+    cchq {settings.SERVER_ENVIRONMENT} run-shell-command elasticsearch "cat /opt/data/elasticsearch*/logs/*es.log | grep '<task_id>.*ReindexResponse'"
```

Comment:
```suggestion
    cchq {settings.SERVER_ENVIRONMENT} run-shell-command elasticsearch "grep '<task_id>.*ReindexResponse' /opt/data/elasticsearch*/logs/*es.log"
```

Reply:
Ahh! I missed it. Thanks for suggestion 🙏